python/lsst/summit/extras/logUtils.py

```diff
@@ -35,6 +34,8 @@ class LogBrowser():
 
```

Comment:
According to [the Python docs](https://docs.python.org/3/tutorial/classes.html#class-definition-syntax) (and my formatter), it looks like it's better to define this class without closing parentheses when empty, i.e.:
```
class LogBrowser:
```

Reply:
Tis a bad habit of mine from the bad old days. I just cleaned up a whole package the other day after Jim pointed that out 🙂